collect_app/src/main/res/values/strings.xml

```diff
@@ -23,6 +23,7 @@
     <string name="cancel">Cancel</string>
     <string name="cancel_loading_form">Cancel</string>
     <string name="cancel_location">Cancel</string>
+    <string name="cancelling">Canceling</string>
```

Comment:
We should consistently use one or the other. Do we usually use U.S. spellings? Canceling, if so?

Reply:
Its is a typo error, I think we use U.S. Spellings.